core/auth/src/main/java/software/amazon/awssdk/auth/credentials/AwsCredentials.java

```diff
@@ -31,7 +31,7 @@
 public interface AwsCredentials {
 
     /**
-     * Retrieve the AWS access key, used to identify the user interacting with AWS.
+     * Retrieve the AWS access key, used to identify the user interacting with AWS.test
```

Comment:
Was put here by mistake?

Reply:
Same as above, this was added to trigger the protocol tests benchmark. Will remove when the PR is close to be merged.